### PR TITLE
created play pause button and tested functionality using local audio …

### PR DESCRIPTION
**Pull request by ToriFortune**
…file

### CODE REVIEW

**Comment by ToriFortune** [comment]
created play pause button and tested functionality using local audio

File: assets/javascript/musicAPI.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+
+
+
+
+// created function to get song by title making an ajax call to the indicated database
+function getSongs(songTitle){
+    // created url variable to call song from returned array from database also making an ajax call. 
+    var url = "https://musicdemons.com/api/v1/song";
+    $.ajax({
```
Comment (apizarras) [inline assets/javascript/musicAPI.js:10]: update 'var' to const or let appropriately

File: assets/javascript/musicAPI.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        url: url, method:"GET"
+    }).then(function(response){
+        //Used filter method to compare selected song title queried from the URL with returned value in the response
+        var thisSong = response.filter(function(song){return song.title === songTitle})[0];
```
Comment (apizarras) [inline assets/javascript/musicAPI.js]: update 'var' to const or let appropriately

Comment (apizarras) [inline assets/javascript/musicAPI.js]: is there an array that you are referencing with the '[0]' that we need to account for?

File: assets/javascript/musicAPI.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+
+// created function to get song by title making an ajax call to the indicated database
+function getSongs(songTitle){
+    // created url variable to call song from returned array from database also making an ajax call. 
+    var url = "https://musicdemons.com/api/v1/song";
+    $.ajax({
+        url: url, method:"GET"
```
Comment (apizarras) [inline assets/javascript/musicAPI.js]: suggest we list each of these on a seperate line to keep code organized

**Review by iaiqbal — APPROVED** [review]
Looks good

**Pull request closed without merging**